docs/en/stack/ml/anomaly-detection/ml-revert-model-snapshot.asciidoc

```diff
@@ -0,0 +1,43 @@
+[role="xpack"]
+[[ml-reverting-model-snapshot]]
+= Reverting to a model snapshot
+
+{ml-docs}/ml-ad-run-jobs.html#ml-ad-model-snapshots[Snapshots of the {ml} model] 
+for each {anomaly-job} are saved frequently to an internal {es} index to ensure 
+resilience. It makes it possible to reset the model to a previous state in case 
+of a system failure or if the model changed significantly due to a one-off 
+event.
+
+. In {kib}, navigate to **{ml-app} > Anomaly Detection > Jobs**.
+. Locate the {anomaly-job} whose model you want to revert in the job table.
+. Open the job details and navigate to the **Model Snapshots** tab.
++
+--
+[role="screenshot"]
+image::images/anomaly-job-model-snapshots.jpg[A screenshot of a job with the Model Snapshots tab opened]
+--
+. Select the snapshot you want to revert from the list and click the **Revert** 
```

Comment:
Not mandatory, but I think this is clearer since "reverting a snapshot" implies you're doing something to the snapshot itself:
```suggestion
. Select a snapshot from the list and click the **Revert** 
```